aptos-move/framework/aptos-stdlib/sources/signature.move

```diff
@@ -177,6 +184,14 @@ module aptos_std::signature {
         signature: vector<u8>
     ): (vector<u8>, bool);
 
+    public fun ed25519_verify_t<T: drop> (signature: vector<u8>, public_key: vector<u8>, data: T): bool {
```

Comment:
This needs comments. For example,  is this function meant to verify only key rotation proofs?

Reply:
for now yes - will add comments soon